pkg/hubble/server/local_observer.go

```diff
@@ -106,13 +112,15 @@ func NewLocalServer(
 	}).Info("Configuring Hubble server")
 
 	s := &LocalObserverServer{
-		log:           logger,
-		ring:          container.NewRing(opts.MaxFlows),
-		events:        make(chan *pb.Payload, opts.MonitorBuffer),
-		stopped:       make(chan struct{}),
-		eventschan:    make(chan *observer.GetFlowsResponse, 100), // option here?
-		payloadParser: payloadParser,
-		opts:          opts,
+		log:              logger,
+		ring:             container.NewRing(opts.MaxFlows),
+		events:           make(chan *pb.Payload, opts.MonitorBuffer),
+		stopped:          make(chan struct{}),
+		eventschan:       make(chan *observer.GetFlowsResponse, 100), // option here?
+		payloadParser:    payloadParser,
+		startTime:        time.Now(),
+		numObservedFlows: 0,
```

Comment:
I don't think this member needs to be explicitly initialized as it's zero-value is 0.